modules/nickserv.cpp

```diff
@@ -93,6 +112,9 @@ class CNickServ : public CModule {
             SetNV("IdentifyCmd", "NICKSERV IDENTIFY {password}");
         }
 
+        CString sTmp;
+        m_bJoinAfterIdentified = (sTmp = GetNV("JoinAfterIdentified")).empty() ? true : sTmp.ToBool();
```

Comment:
I don't think default should be true...
Nickserv module is too fragile, and if user just stops joining any channel, how is he supposed to know what happened?